src/main/java/uk/gov/companieshouse/company/metrics/repository/CompanyMetricsRepository.java

```diff
@@ -0,0 +1,7 @@
+package uk.gov.companieshouse.company.metrics.repository;
+
+import org.springframework.data.mongodb.repository.MongoRepository;
+import uk.gov.companieshouse.company.metrics.model.CompanyMetricsDocument;
```

Comment:
Probably good have a unit test for this repository?

Reply:
done